### PR TITLE
Bump tektoncd/triggers to v0.5.0

### LINKED ISSUE #984: Bump pipeline and triggers dep
We should bump the tektoncd/pipeline  to 0.12.x and tektoncd/triggers to 0.5.x

/kind feature
/assign

### PR DESCRIPTION
**Pull request by piyush-garg**
This will bump tektoncd/triggers dependency
to version v0.5.0 in vendor

Fix #984

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [x] Run the code checkers with `make check`
- [x] Regenerate the manpages, docs and go formatting with `make generated`
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/cli/blob/master/CONTRIBUTING.md)
for more details._

# Release Notes

```
Bump tektoncd/triggers to v0.5.0
```

### CODE REVIEW

**Comment by piyush-garg** [comment]
cc @chmouel @savitaashture @khrm

File: pkg/cmd/eventlistener/describe.go [hunk 1]
```diff
@@ -68,9 +68,9 @@ const describeTemplate = `{{decorate "bold" "Name"}}:	{{ .EventListener.Name }}
  {{decorate "bullet" $v.Name }}
 {{ " " }}
 {{- end }}
- BINDING NAME	KIND	APIVERSION
+ BINDING REF	KIND	APIVERSION
```
Comment (savitaashture) [inline pkg/cmd/eventlistener/describe.go:71]: Binding can have both `name`,  `ref` as shown [here](https://github.com/tektoncd/triggers/blob/master/examples/eventlisteners/cel-eventlistener-interceptor.yaml)
with the current change if EventListener  have both `name` and `ref` as shown [here](https://github.com/tektoncd/triggers/blob/master/examples/eventlisteners/cel-eventlistener-interceptor.yaml) then describe doesn't work properly for `name`

Comment (piyush-garg) [reply pkg/cmd/eventlistener/describe.go:71]: Yes, for that I have created https://github.com/tektoncd/cli/issues/1004 to support this.

**Pull request merged**